main.go

```diff
@@ -81,6 +95,11 @@ func main() {
 		go worker(wg, ch)
 	}
 	wg.Wait()
+	if opts.EnableHibernate {
+		for {
+			time.Sleep(time.Hour)
```

Comment:
You can actually just create a channel and block on it IIRC (to avoid the wake-up every hour ;) )